src/test/java/org/jfrog/hudson/pipeline/integration/CommonITestsPipeline.java

```diff
@@ -278,6 +278,26 @@ void mavenTest(String buildName) throws Exception {
         }
     }
 
+    void mavenWrapperTest(String buildName) throws Exception {
+        Set<String> expectedArtifacts = Sets.newHashSet("multi-3.7-SNAPSHOT.pom");
+        String buildNumber = "3";
+        try {
+            runPipeline("mavenWrapper", false);
+            Build buildInfo = getBuildInfo(buildInfoClient, buildName, buildNumber);
+            assertFilteredProperties(buildInfo);
```

Comment:
It looks like the assertions section in this test method, can me merged with the assertions part in the ```mavenWrapperTest``` method.